lib/module.js

```diff
@@ -152,14 +144,15 @@ Module._findPath = function(request, paths, isMain) {
     return false;
   }
 
-  const cacheKey = JSON.stringify({request: request, paths: paths});
-  if (Module._pathCache[cacheKey]) {
-    return Module._pathCache[cacheKey];
-  }
+  var cacheKey = request + '\x00' +
+                (paths.length === 1 ? paths[0] : paths.join('\x00'));
+  var entry = Module._pathCache[cacheKey];
```

Comment:
It's too bad that we can't use the github emoji thumbsup on individual changes ;-)